question16/main.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const createElementWithClassName = (element, name) => {
+  const createdElement = document.createElement(element);
+  createdElement.classList.add(name);
+  return createdElement;
+};
+
+const newsWrapper = document.getElementById('js-newsWrapper');
+const tabsGroup = document.getElementById('js-tabs');
+const contentsWrapper = document.createElement("div");
+contentsWrapper.id = "js-contentsWrapper";
+contentsWrapper.classList.add("contentsWrapper");
+
+
+//Get json data
+async function callApi() {
+  try {
+    const res = await fetch('https://myjson.dit.upm.es/api/bins/an1r');
+    const json = await res.json();
+    return json.data;
+  }
+  catch(error) {
+    tabsGroup.textContent = "データが取得できませんでした";
```

Comment:
You put the error text inside the `ul` tag, but it's not a list, so it seems better to change the tag.

Reply:
Thanks for the review! I changed the tag specification from ul to parent element. Please confirm.